resources/assets/js/back/app.js

```diff
@@ -15,27 +15,23 @@ import { query } from 'spatie-dom';
 if (query('blender-media')) {
     require.ensure([], () => {
         require('./modules/media').default();
-    }, 'back.media');
+    });
 }
 
 if (query('blender-content-blocks')) {
     require.ensure([], () => {
         require('./modules/contentBlocks').default();
-    }, 'back.contentBlocks');
+    });
 }
 
 if (query('blender-chart')) {
     require.ensure([], () => {
         require('./modules/chart').default();
-    }, 'back.chart');
+    });
 }
 
-/*
-*  uncomment when redactor is installed
-*
 if (query('[data-editor]')) {
```

Comment:
Not commented out by default anymore? It's easier for us now, but mix fails on first run for newbies